pyproject.toml

```diff
@@ -0,0 +1,190 @@
+[tool.poetry]
+name = "screenpy_examples"
+version = "0.1.0"
+description = "Screenpy Examples"
+authors = ["Perry Goy <perry.goy@gmail.com>"]
+maintainers = ["Gabe Langton", "Marcel Wilson"]
+license = "MIT"
+repository = "https://github.com/ScreenPyHQ/screenpy_examples"
+#documentation = "https://screenpy-docs.readthedocs.io"  ????
+readme = "README.md"
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Quality Assurance",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Software Development :: Testing :: BDD",
+    "Natural Language :: English",
+    "License :: OSI Approved :: MIT License",
+]
+
+
+[tool.poetry.dependencies]
+python = "^3.11"
+pytest = "*"
+screenpy = "*"
+screenpy-selenium = { git = "git@github.com:ScreenPyHQ/screenpy_selenium.git", branch = "trunk"}
+#screenpy-selenium = "*"
+screenpy-appium = "*"
+screenpy-playwright = "*"
+screenpy-requests = "*"
+screenpy-adapter-allure = "^4.0.2"
+pyderman = "^3.3.2"
+semantic-version = "^2.10.0"
+
+#tox = {version = "*", optional = true}
+mypy = {version = "*", optional = true}
+black = {version = "*", optional = true}
+isort = {version = "*", optional = true}
+flake8 = {version = "*", optional = true}
+pylint = {version = "*", optional = true}
+coverage = {version = "*", optional = true}
+ruff = {version = "*", optional = true}
+setuptools = "*"
+types-requests = {version = "^2.31.0.2", optional = true}
+
+
+[tool.poetry.extras]
+dev = [
+    "pre-commit",
+#    "tox",
+    "pylint",
+    "mypy",
+    "black",
+    "isort",
+    "flake8",
+    "coverage",
+    "ruff",
+    "types-requests",
+]
+
+
+[tool.isort]
+line_length = 88
+multi_line_output = 3
+include_trailing_comma = "True"
+use_parentheses = "True"
+combine_as_imports = "True"
+extend_skip = [".idea"]
+
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+
+[tool.ruff]
+# Enable the pycodestyle (`E`) and Pyflakes (`F`) rules by default.
+# Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
+# McCabe complexity (`C901`) by default.
```

Comment:
It looks like there are a lot of commented-out lines and lines from the default `ruff` config, should we have a chat about what rules to use and which to ignore?